robottelo/ui/locators/base.py

```diff
@@ -1687,8 +1687,9 @@
         By.XPATH, "//td/a[contains(., '%s')]/following::td/div/"
                   "button[contains(@class, 'dropdown-toggle')]"),
     "contentviews.remove_ver": (
-        By.XPATH, ("//td/a[contains(., '%s')]/following::td//"
-                   "a[contains(@ng-click, 'version-deletion')]")),
+        By.XPATH,
+        ("//td/a[contains(., '%s')]/following::td[@class='col-sm-2'][1]"
```

Comment:
Of course, it is going to take first element from the list according to Selenium paradigm, but I hate when my locator returns more than one element